covalent_dispatcher/_service/app.py

```diff
@@ -85,3 +91,63 @@ def cancel() -> Response:
 def db_path() -> Response:
     db_path = DispatchDB()._dbpath
     return jsonify(db_path)
+
+
+@bp.route("/result/<dispatch_id>", methods=["GET"])
+def get_result(dispatch_id) -> Response:
+    args = request.args
+    wait = args.get("wait", default=False, type=lambda v: v.lower() == "true")
+    status_only = args.get("status_only", default=False, type=lambda v: v.lower() == "true")
+    while True:
+        with Session(DispatchDB()._get_data_store().engine) as session:
+            lattice_record = (
+                session.query(Lattice).where(Lattice.dispatch_id == dispatch_id).first()
+            )
+        try:
+            if not lattice_record:
+                return (
+                    jsonify(
+                        {"message": f"The requested dispatch ID {dispatch_id} was not found."}
+                    ),
+                    404,
+                )
+            elif not wait:
+                output = {
+                    "id": dispatch_id,
+                    "status": lattice_record.status,
+                }
+                if not status_only:
+                    output["result"] = codecs.encode(
+                        pickle.dumps(result_from(lattice_record)), "base64"
+                    ).decode()
+                return jsonify(output)
+            elif lattice_record.status in [

```

Comment:
can the `elif not wait` and `elif lattice_record.status` be combined to a single `else`... seems like we are performing the exact same action in the body in both cases?